ui/src/app/views/quick-tour-modal/component.css

```diff
@@ -0,0 +1,165 @@
+.main {
+  height: 79.23%;
+  width: 64.92%;
+  border-radius: 8px;
+  background-color: #262262;
+  box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+  position: absolute;
+  left: 18%;
+  top: 5%;
+}
+
+.title {
+  color: #FFFFFF;
+  margin-top: 2%;
+  margin-left: 3%;
+  font-size: 21px;
+  width: 100%;
+}
+
+.intro {
+  color: #FFFFFF;
+  font-size: 48px;
+  width: 100%;
+  margin-top: 3%;
+  margin-left: 3%;
+  font-weight: 600;
+}
+
+.breadcrumbs {
+  width: 100%;
+  margin-top: 5%;
+  padding-left: 25%;
```

Comment:
Nit: I think you can do this with `justify-content: center` and that removes the need to have padding/guarantees the content is in the center of the modal